### ISSUE TITLE
Incorrect note durations in some triplets

### ISSUE THREAD

**Issue by jtna** [paulrosen/abcjs#670]
There is corner-case bug in calculating the note durations in triplets, if `p != r` in `(p:q:r` syntax (which means 'put p notes into the time of q for the next r notes').

Given this abc string:
```
M: 1/4
L: 1/4
Q: 1/4=60
(3A/A/A/ | (3:2:2A/A | (3:2:2AA/ |
```
<img width="210" alt="triplets" src="https://user-images.githubusercontent.com/80085039/116183661-2391cc80-a6d3-11eb-8b3f-8a26ba8361a4.png">

The note durations for the first three bars are:
```
1) 0.083333, 0.083333, 0.083334: correct
2) 0.083333, 0.166667: correct
3) 0.166667, 0.333333: INCORRECT
```

The problem is that, when calculating the tripletDurationTotal in abc_midi_sequencer.js, the first note in the triplet is used as a base. So tripletDurationTotal for bar 3 ends up being twice as big as what it should be.

I believe the simplest solution here is to traverse the notes in the triplet to get the tripletDurationTotal, if `p != r`.

I'll submit a pull request for this. Please let me know if you believe there's a better way of fixing this.

**Comment by paulrosen** [comment]
Will be fixed in 6.0.0-beta.34

**Comment by paulrosen** [comment]
Fixed in version 6.0.0-beta.35
